app/views/partials/maps/_main.html.erb

```diff
@@ -1,10 +1,9 @@
 <div class="map--main">
+  <v-map-header title="<%= @main_map[:title] %>"></v-map-header>
+  <v-map></v-map>
   <v-map-filters
     title="<%= @main_map[:title] %>"
     :overlays="<%= @main_map[:overlays].to_json %>"
     :disclaimer="<%= @main_map[:disclaimer].to_json %>"
-  >
-    <v-map-p-a-search></v-map-p-a-search>
```

Comment:
Do you plan to add this back in?

Reply:
Isn't it used via the Filters component(?)

Reply:
I was initially passing it in as a slot, in the hope that it could be a bit separate to the more reusable/generic map component. I don't know how feasible that separation will be though, as I'm not too sure how the search will work yet...

Reply:
Can I leave it out until such time we figure this out then?

Reply:
Sure - in that case I would remove it from vue.js (or application.js... I forget the file name) as well as it isn't used anywhere else. Thanks!